sql/core/src/test/resources/sql-tests/inputs/count.sql

```diff
@@ -19,11 +19,11 @@ SELECT
 FROM testData;
 
 -- count with multiple expressions
-SELECT count(a, b), count(b, a), count(testData.*) FROM testData;
+SELECT count(a, b), count(b, a), count(testData.a, testData.b) FROM testData;
```

Comment:
how about `count(testData.*, testData.*)`

Reply:
sure, I changed this in both count.sql and udf-count.sql.